CMake config file is placed in lib64 instead of lib

- [x] I read [the conda-forge documentation](https://conda-forge.org/docs/user/introduction.html#how-can-i-install-packages-from-conda-forge) and could not find the solution for my problem there.

Issue:

The CMake config file for backward is placed in `lib64` instead of `lib` in the conda prefix, so cmake fails to find it (in at least my case). This is a regression from the 1.4 package. (No other conda packages I am using install anything into `lib64` and I was not able to find any mention of it online. So I think only `lib` should be used in conda packages.)

I get the typical cmake error:
<details>

```
CMake Error at libsupport/CMakeLists.txt:76 (find_package):
  By not providing "FindBackward.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "Backward",
  but CMake did not find one.

  Could not find a package configuration file provided by "Backward" with any
  of the following names:

    BackwardConfig.cmake
    backward-config.cmake

  Add the installation prefix of "Backward" to CMAKE_PREFIX_PATH or set
  "Backward_DIR" to a directory containing one of the above files.  If
  "Backward" provides a separate development package or SDK, be sure it has
  been installed.
```
</details>

<br/>
Environment (<code>conda list</code>):
<details>

```
$ conda list
# packages in environment at /home/amp/LocalInstalls/conda/envs/katana-test:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                 conda_forge    conda-forge
_openmp_mutex             4.5                       1_gnu    conda-forge
abseil-cpp                20210324.2           h9c3ff4c_0    conda-forge
alabaster                 0.7.12                     py_0    conda-forge
alsa-lib                  1.2.3                h516909a_0    conda-forge
anyio                     3.3.0            py38h578d9bd_0    conda-forge
appdirs                   1.4.4              pyh9f0ad1d_0    conda-forge
argon2-cffi               20.1.0           py38h497a2fe_2    conda-forge
arrow-cpp                 4.0.1           py38ha36b58d_8_cpu    conda-forge
astroid                   2.5.6            py38h578d9bd_0    conda-forge
atk-1.0                   2.36.0               h3371d22_4    conda-forge
attrs                     20.3.0             pyhd3deb0d_0    conda-forge
aws-c-cal                 0.5.11               h95a6274_0    conda-forge
aws-c-common              0.6.2                h7f98852_0    conda-forge
aws-c-event-stream        0.2.7               h3541f99_13    conda-forge
aws-c-io                  0.10.5               hfb6a706_0    conda-forge
aws-checksums             0.1.11               ha31a3da_7    conda-forge
aws-sdk-cpp               1.8.186              hb4091e7_3    conda-forge
babel                     2.9.1              pyh44b312d_0    conda-forge
backcall                  0.2.0              pyh9f0ad1d_0    conda-forge
backports                 1.0                        py_2    conda-forge
backports.functools_lru_cache 1.6.4              pyhd8ed1ab_0    conda-forge
backward-cpp              1.6                  h9c3ff4c_1    conda-forge
beautifulsoup4            4.9.3              pyhb0f4dca_0    conda-forge
benchmark                 1.5.6                h9c3ff4c_0    conda-forge
binutils                  2.36.1               hdd6e379_2    conda-forge
binutils_impl_linux-64    2.36.1               h193b22a_2    conda-forge
binutils_linux-64         2.36                 hf3e587d_0    conda-forge
black                     19.10b0                  py38_0    conda-forge
bleach                    4.1.0              pyhd8ed1ab_0    conda-forge
bokeh                     2.3.3            py38h578d9bd_0    conda-forge
boost-cpp                 1.77.0               h312852a_0    conda-forge
bottle                    0.12.18                    py_0    conda-forge
breathe                   4.30.0             pyhd8ed1ab_0    conda-forge
brotlipy                  0.7.0           py38h497a2fe_1001    conda-forge
bzip2                     1.0.8                h7f98852_4    conda-forge
c-ares                    1.17.2               h7f98852_0    conda-forge
c-compiler                1.2.0                h7f98852_0    conda-forge
ca-certificates           2021.5.30            ha878542_0    conda-forge
cairo                     1.16.0            h6cf1ce9_1008    conda-forge
certifi                   2021.5.30        py38h578d9bd_0    conda-forge
cffi                      1.14.6           py38ha65f79e_0    conda-forge
chardet                   3.0.4           py38h924ce5b_1008    conda-forge
click                     8.0.1            py38h578d9bd_0    conda-forge
cloudpickle               1.6.0                      py_0    conda-forge
cmake                     3.21.2               h8897547_0    conda-forge
colorama                  0.4.4              pyh9f0ad1d_0    conda-forge
conan                     1.39.0             pyhd8ed1ab_0    conda-forge
conda                     4.10.3           py38h578d9bd_0    conda-forge
conda-build               3.21.4           py38h578d9bd_0    conda-forge
conda-package-handling    1.7.3            py38h497a2fe_0    conda-forge
conda-verify              3.1.1           py38h578d9bd_1003    conda-forge
coverage                  5.5              py38h497a2fe_0    conda-forge
cryptography              3.4.7            py38ha5dfef3_0    conda-forge
cxx-compiler              1.2.0                h4bd325d_0    conda-forge
cycler                    0.10.0                     py_2    conda-forge
cython                    0.29.24          py38h709712a_0    conda-forge
cytoolz                   0.11.0           py38h497a2fe_3    conda-forge
dask                      2021.8.1           pyhd8ed1ab_0    conda-forge
dask-core                 2021.8.1           pyhd8ed1ab_0    conda-forge
dbus                      1.13.6               h48d8840_2    conda-forge
debugpy                   1.4.1            py38h709712a_0    conda-forge
decorator                 4.4.2                      py_0    conda-forge
defusedxml                0.7.1              pyhd8ed1ab_0    conda-forge
deprecated                1.2.12             pyh44b312d_0    conda-forge
deprecation               2.0.6                      py_0    conda-forge
distributed               2021.8.1         py38h578d9bd_0    conda-forge
distro                    1.5.0              pyh9f0ad1d_0    conda-forge
docutils                  0.16             py38h578d9bd_3    conda-forge
donfig                    0.6.0              pyhd8ed1ab_0    conda-forge
doxygen                   1.9.1                hb166930_1    conda-forge
eigen                     3.4.0                h4bd325d_0    conda-forge
elfutils                  0.185                he364ef2_0    conda-forge
entrypoints               0.3             py38h32f6830_1002    conda-forge
expat                     2.4.1                h9c3ff4c_0    conda-forge
fasteners                 0.16               pyhd8ed1ab_0    conda-forge
filelock                  3.0.12             pyh9f0ad1d_0    conda-forge
fmt                       8.0.1                h4bd325d_0    conda-forge
font-ttf-dejavu-sans-mono 2.37                 hab24e00_0    conda-forge
font-ttf-inconsolata      3.000                h77eed37_0    conda-forge
font-ttf-source-code-pro  2.038                h77eed37_0    conda-forge
font-ttf-ubuntu           0.83                 hab24e00_0    conda-forge
fontconfig                2.13.1            hba837de_1005    conda-forge
fonts-conda-ecosystem     1                             0    conda-forge
fonts-conda-forge         1                             0    conda-forge
freetype                  2.10.4               h0708190_1    conda-forge
fribidi                   1.0.10               h36c2ea0_0    conda-forge
fsspec                    2021.7.0           pyhd8ed1ab_0    conda-forge
future                    0.18.2           py38h578d9bd_3    conda-forge
gcc_impl_linux-64         9.4.0                h03d3576_8    conda-forge
gcc_linux-64              9.4.0                h391b98a_0    conda-forge
gdk-pixbuf                2.42.6               h04a7f16_0    conda-forge
gettext                   0.19.8.1          h0b5b191_1005    conda-forge
gflags                    2.2.2             he1b5a44_1004    conda-forge
giflib                    5.2.1                h36c2ea0_2    conda-forge
gitdb                     4.0.7              pyhd8ed1ab_0    conda-forge
gitpython                 3.1.18             pyhd8ed1ab_0    conda-forge
glib                      2.68.4               h9c3ff4c_0    conda-forge
glib-tools                2.68.4               h9c3ff4c_0    conda-forge
glob2                     0.7                        py_0    conda-forge
glog                      0.5.0                h48cff8f_0    conda-forge
gnutls                    3.6.13               h85f3911_1    conda-forge
graphblas                 5.1.5                h9c3ff4c_0    conda-forge
graphite2                 1.3.13            h58526e2_1001    conda-forge
graphviz                  2.49.0               h85b4f2f_0    conda-forge
grblas                    1.3.10           py38h578d9bd_0    conda-forge
grpc-cpp                  1.39.1               hf1f433d_0    conda-forge
gst-plugins-base          1.18.4               hf529b03_2    conda-forge
gstreamer                 1.18.4               h76c114f_2    conda-forge
gtk2                      2.24.33              h539f30e_1    conda-forge
gts                       0.7.6                h64030ff_2    conda-forge
gxx_impl_linux-64         9.4.0                h03d3576_8    conda-forge
gxx_linux-64              9.4.0                h0316aca_0    conda-forge
harfbuzz                  2.9.0                h83ec7ef_0    conda-forge
heapdict                  1.0.1                      py_0    conda-forge
icu                       68.1                 h58526e2_0    conda-forge
idna                      2.10               pyh9f0ad1d_0    conda-forge
imagesize                 1.2.0                      py_0    conda-forge
importlib-metadata        4.8.1            py38h578d9bd_0    conda-forge
importlib_metadata        4.8.1                hd8ed1ab_0    conda-forge
importlib_resources       5.2.2              pyhd8ed1ab_0    conda-forge
iniconfig                 1.1.1              pyh9f0ad1d_0    conda-forge
ipykernel                 6.3.0            py38he5a9106_0    conda-forge
ipython                   7.27.0           py38he5a9106_0    conda-forge
ipython_genutils          0.2.0                      py_1    conda-forge
ipywidgets                7.6.3              pyhd3deb0d_0    conda-forge
isort                     5.9.3              pyhd8ed1ab_0    conda-forge
jbig                      2.1               h7f98852_2003    conda-forge
jedi                      0.18.0           py38h578d9bd_2    conda-forge
jemalloc                  5.2.1                h9c3ff4c_5    conda-forge
jinja2                    2.11.3             pyh44b312d_0    conda-forge
jpeg                      9d                   h36c2ea0_0    conda-forge
jsonschema                3.2.0            py38h32f6830_1    conda-forge
jupyter                   1.0.0            py38h578d9bd_6    conda-forge
jupyter-server-mathjax    0.2.3              pyhd8ed1ab_0    conda-forge
jupyter_client            6.1.12             pyhd8ed1ab_0    conda-forge
jupyter_console           6.4.0              pyhd8ed1ab_0    conda-forge
jupyter_core              4.7.1            py38h578d9bd_0    conda-forge
jupyter_server            1.10.2             pyhd8ed1ab_0    conda-forge
jupyterlab_widgets        1.0.0              pyhd8ed1ab_1    conda-forge
kernel-headers_linux-64   2.6.32              he073ed8_14    conda-forge
kiwisolver                1.3.2            py38h1fd1430_0    conda-forge
krb5                      1.19.2               hcc1bbae_0    conda-forge
lazy-object-proxy         1.6.0            py38h497a2fe_0    conda-forge
lcms2                     2.12                 hddcbb42_0    conda-forge
ld_impl_linux-64          2.36.1               hea4e1c9_2    conda-forge
lerc                      2.2.1                h9c3ff4c_0    conda-forge
libarchive                3.5.2                hccf745f_0    conda-forge
libblas                   3.9.0           11_linux64_openblas    conda-forge
libbrotlicommon           1.0.9                h7f98852_5    conda-forge
libbrotlidec              1.0.9                h7f98852_5    conda-forge
libbrotlienc              1.0.9                h7f98852_5    conda-forge
libcblas                  3.9.0           11_linux64_openblas    conda-forge
libclang                  11.1.0          default_ha53f305_1    conda-forge
libcurl                   7.78.0               h2574ce0_0    conda-forge
libdeflate                1.7                  h7f98852_5    conda-forge
libedit                   3.1.20191231         he28a2e2_2    conda-forge
libev                     4.33                 h516909a_1    conda-forge
libevent                  2.1.10               hcdb4288_3    conda-forge
libffi                    3.3                  h58526e2_2    conda-forge
libgcc-devel_linux-64     9.4.0                hd854feb_8    conda-forge
libgcc-ng                 11.1.0               hc902ee8_8    conda-forge
libgd                     2.3.2                h78a0170_0    conda-forge
libgfortran-ng            11.1.0               h69a702a_8    conda-forge
libgfortran5              11.1.0               h6c583b3_8    conda-forge
libglib                   2.68.4               h3e27bee_0    conda-forge
libgomp                   11.1.0               hc902ee8_8    conda-forge
libiconv                  1.16                 h516909a_0    conda-forge
liblapack                 3.9.0           11_linux64_openblas    conda-forge
liblief                   0.11.5               h9c3ff4c_0    conda-forge
libllvm11                 11.1.0               hf817b99_2    conda-forge
libmicrohttpd             0.9.73               h7f98852_0    conda-forge
libnghttp2                1.43.0               h812cca2_0    conda-forge
libogg                    1.3.4                h7f98852_1    conda-forge
libopenblas               0.3.17          pthreads_h8fe5266_1    conda-forge
libopus                   1.3.1                h7f98852_1    conda-forge
libpng                    1.6.37               h21135ba_2    conda-forge
libpq                     13.3                 hd57d9b9_0    conda-forge
libprotobuf               3.16.0               h780b84a_0    conda-forge
librsvg                   2.50.7               hc3c00ef_0    conda-forge
libsanitizer              9.4.0                h79bfe98_8    conda-forge
libsodium                 1.0.18               h36c2ea0_1    conda-forge
libssh2                   1.10.0               ha56f1ee_0    conda-forge
libstdcxx-devel_linux-64  9.4.0                hd854feb_8    conda-forge
libstdcxx-ng              11.1.0               h56837e0_8    conda-forge
libthrift                 0.14.2               he6d91bd_1    conda-forge
libtiff                   4.3.0                hf544144_1    conda-forge
libtool                   2.4.6             h58526e2_1007    conda-forge
libutf8proc               2.6.1                h7f98852_0    conda-forge
libuuid                   2.32.1            h7f98852_1000    conda-forge
libuv                     1.42.0               h7f98852_0    conda-forge
libvorbis                 1.3.7                h9c3ff4c_0    conda-forge
libwebp                   1.2.1                h3452ae3_0    conda-forge
libwebp-base              1.2.1                h7f98852_0    conda-forge
libxcb                    1.13              h7f98852_1003    conda-forge
libxkbcommon              1.0.3                he3ba5ed_0    conda-forge
libxml2                   2.9.12               h72842e0_0    conda-forge
llvm-meta                 7.0.0                         0    conda-forge
llvm-tools                11.1.0               hf817b99_2    conda-forge
llvmdev                   11.1.0               hf817b99_2    conda-forge
llvmlite                  0.37.0           py38h4630a5e_0    conda-forge
locket                    0.2.0                      py_2    conda-forge
lz4-c                     1.9.3                h9c3ff4c_1    conda-forge
lzo                       2.10              h516909a_1000    conda-forge
make                      4.3                  hd18ef5c_1    conda-forge
markupsafe                2.0.1            py38h497a2fe_0    conda-forge
matplotlib-base           3.4.3            py38hf4fb855_0    conda-forge
matplotlib-inline         0.1.2              pyhd8ed1ab_2    conda-forge
mccabe                    0.6.1                      py_1    conda-forge
metagraph                 0.2.6                g69d4090_0    metagraph
mistune                   0.8.4           py38h497a2fe_1004    conda-forge
monotonic                 1.5                        py_0    conda-forge
more-itertools            8.8.0              pyhd8ed1ab_0    conda-forge
msgpack-python            1.0.2            py38h1fd1430_1    conda-forge
mypy_extensions           0.4.3            py38h578d9bd_3    conda-forge
mysql-common              8.0.25               ha770c72_2    conda-forge
mysql-libs                8.0.25               hfa10184_2    conda-forge
nbconvert                 5.6.1            py38h32f6830_1    conda-forge
nbdime                    3.1.0              pyhd8ed1ab_0    conda-forge
nbformat                  5.1.3              pyhd8ed1ab_0    conda-forge
ncurses                   6.2                  h58526e2_4    conda-forge
nest-asyncio              1.5.1              pyhd8ed1ab_0    conda-forge
nettle                    3.6                  he412f7d_0    conda-forge
networkx                  2.6.2              pyhd8ed1ab_0    conda-forge
ninja                     1.10.2               h4bd325d_0    conda-forge
nlohmann_json             3.10.2               h9c3ff4c_0    conda-forge
notebook                  6.4.3              pyha770c72_0    conda-forge
nspr                      4.30                 h9c3ff4c_0    conda-forge
nss                       3.69                 hb5efdd6_0    conda-forge
numba                     0.54.0           py38h4bf6c61_0    conda-forge
numpy                     1.20.3           py38h9894fe3_1    conda-forge
olefile                   0.46               pyh9f0ad1d_1    conda-forge
openblas                  0.3.17          pthreads_h4748800_1    conda-forge
openjpeg                  2.4.0                hb52868f_1    conda-forge
openmp                    7.0.0                h2d50403_0    conda-forge
openssl                   1.1.1k               h7f98852_1    conda-forge
orc                       1.6.10               h58a87f1_0    conda-forge
packaging                 21.0               pyhd8ed1ab_0    conda-forge
pandas                    1.3.2            py38h43a58ef_0    conda-forge
pandoc                    2.14.2               h7f98852_0    conda-forge
pandocfilters             1.4.2                      py_1    conda-forge
pango                     1.48.9               hb8ff022_0    conda-forge
parquet-cpp               1.5.1                         1    conda-forge
parso                     0.8.2              pyhd8ed1ab_0    conda-forge
partd                     1.2.0              pyhd8ed1ab_0    conda-forge
patch-ng                  1.17.4             pyh9f0ad1d_0    conda-forge
patchelf                  0.13                 h58526e2_0    conda-forge
pathspec                  0.9.0              pyhd8ed1ab_0    conda-forge
pcre                      8.45                 h9c3ff4c_0    conda-forge
pexpect                   4.8.0            py38h32f6830_1    conda-forge
pickleshare               0.7.5           py38h32f6830_1002    conda-forge
pillow                    8.3.1            py38h8e6f84c_0    conda-forge
pip                       21.2.4             pyhd8ed1ab_0    conda-forge
pixman                    0.40.0               h36c2ea0_0    conda-forge
pkginfo                   1.7.1              pyhd8ed1ab_0    conda-forge
pluggy                    0.13.1           py38h578d9bd_4    conda-forge
pluginbase                1.0.0                      py_0    conda-forge
prometheus_client         0.11.0             pyhd8ed1ab_0    conda-forge
prompt-toolkit            3.0.20             pyha770c72_0    conda-forge
prompt_toolkit            3.0.20               hd8ed1ab_0    conda-forge
psutil                    5.8.0            py38h497a2fe_1    conda-forge
pthread-stubs             0.4               h36c2ea0_1001    conda-forge
ptyprocess                0.7.0              pyhd3deb0d_0    conda-forge
py                        1.10.0             pyhd3deb0d_0    conda-forge
py-lief                   0.11.5           py38h709712a_0    conda-forge
pyarrow                   4.0.1           py38h8752a10_8_cpu    conda-forge
pycosat                   0.6.3           py38h497a2fe_1006    conda-forge
pycparser                 2.20               pyh9f0ad1d_2    conda-forge
pydata-sphinx-theme       0.6.3              pyhd8ed1ab_0    conda-forge
pygithub                  1.54.1             pyhd3deb0d_1    conda-forge
pygments                  2.10.0             pyhd8ed1ab_0    conda-forge
pyjwt                     1.7.1                      py_0    conda-forge
pylint                    2.8.3              pyhd8ed1ab_0    conda-forge
pyopenssl                 20.0.1             pyhd8ed1ab_0    conda-forge
pyparsing                 2.4.7              pyh9f0ad1d_0    conda-forge
pyqt                      5.12.3           py38h578d9bd_7    conda-forge
pyqt-impl                 5.12.3           py38h7400c14_7    conda-forge
pyqt5-sip                 4.19.18          py38h709712a_7    conda-forge
pyqtchart                 5.12             py38h7400c14_7    conda-forge
pyqtwebengine             5.12.1           py38h7400c14_7    conda-forge
pyrsistent                0.17.3           py38h497a2fe_2    conda-forge
pysocks                   1.7.1            py38h578d9bd_3    conda-forge
pytest                    6.2.4            py38h578d9bd_0    conda-forge
pytest-notebook           0.6.1              pyhd8ed1ab_2    conda-forge
python                    3.8.10          h49503c6_1_cpython    conda-forge
python-dateutil           2.8.2              pyhd8ed1ab_0    conda-forge
python-graphviz           0.17               pyhaef67bd_0    conda-forge
python-libarchive-c       3.1              py38h578d9bd_0    conda-forge
python-louvain            0.15               pyhd3deb0d_0    conda-forge
python-node-semver        0.6.1                      py_0    conda-forge
python-suitesparse-graphblas 5.1.5.1          py38h6c62de6_0    conda-forge
python_abi                3.8                      2_cp38    conda-forge
pytz                      2021.1             pyhd8ed1ab_0    conda-forge
pyyaml                    5.4.1            py38h497a2fe_1    conda-forge
pyzmq                     22.2.1           py38h2035c66_0    conda-forge
qt                        5.12.9               hda022c4_4    conda-forge
qtconsole                 5.1.1              pyhd8ed1ab_0    conda-forge
qtpy                      1.10.0             pyhd8ed1ab_0    conda-forge
re2                       2021.08.01           h9c3ff4c_0    conda-forge
readline                  8.1                  h46c0cb4_0    conda-forge
regex                     2021.8.28        py38h497a2fe_0    conda-forge
requests                  2.24.0             pyh9f0ad1d_0    conda-forge
requests-unixsocket       0.2.0                      py_0    conda-forge
rhash                     1.4.1                h7f98852_0    conda-forge
ripgrep                   13.0.0               habb4d0f_0    conda-forge
ruamel_yaml               0.15.80         py38h497a2fe_1004    conda-forge
s2n                       1.0.10               h9b69904_0    conda-forge
scipy                     1.7.1            py38h56a6a73_0    conda-forge
send2trash                1.8.0              pyhd8ed1ab_0    conda-forge
setuptools                57.4.0           py38h578d9bd_0    conda-forge
shellcheck                0.7.2                ha770c72_1    conda-forge
six                       1.15.0             pyh9f0ad1d_0    conda-forge
smmap                     3.0.5              pyh44b312d_0    conda-forge
snappy                    1.1.8                he1b5a44_3    conda-forge
sniffio                   1.2.0            py38h578d9bd_1    conda-forge
snowballstemmer           2.1.0              pyhd8ed1ab_0    conda-forge
sortedcontainers          2.4.0              pyhd8ed1ab_0    conda-forge
soupsieve                 2.0.1            py38h32f6830_0    conda-forge
sphinx                    3.5.4              pyh44b312d_0    conda-forge
sphinx-tabs               2.1.0              pyhd8ed1ab_0    conda-forge
sphinxcontrib-applehelp   1.0.2                      py_0    conda-forge
sphinxcontrib-devhelp     1.0.2                      py_0    conda-forge
sphinxcontrib-htmlhelp    2.0.0              pyhd8ed1ab_0    conda-forge
sphinxcontrib-jsmath      1.0.1                      py_0    conda-forge
sphinxcontrib-qthelp      1.0.3                      py_0    conda-forge
sphinxcontrib-serializinghtml 1.1.5              pyhd8ed1ab_0    conda-forge
sqlite                    3.36.0               h9cd32fc_0    conda-forge
sysroot_linux-64          2.12                he073ed8_14    conda-forge
tblib                     1.7.0              pyhd8ed1ab_0    conda-forge
terminado                 0.11.1           py38h578d9bd_0    conda-forge
testpath                  0.5.0              pyhd8ed1ab_0    conda-forge
tk                        8.6.11               h21135ba_0    conda-forge
toml                      0.10.2             pyhd8ed1ab_0    conda-forge
toolz                     0.11.1                     py_0    conda-forge
tornado                   6.1              py38h497a2fe_1    conda-forge
tqdm                      4.62.2             pyhd8ed1ab_0    conda-forge
traitlets                 5.0.5                      py_0    conda-forge
typed-ast                 1.4.3            py38h497a2fe_0    conda-forge
typing_extensions         3.10.0.0           pyha770c72_0    conda-forge
urllib3                   1.25.11                    py_0    conda-forge
wcwidth                   0.2.5              pyh9f0ad1d_2    conda-forge
webencodings              0.5.1                      py_1    conda-forge
websocket-client          0.57.0           py38h578d9bd_4    conda-forge
websockets                9.1              py38h497a2fe_0    conda-forge
wheel                     0.37.0             pyhd8ed1ab_1    conda-forge
widgetsnbextension        3.5.1            py38h578d9bd_4    conda-forge
wrapt                     1.12.1           py38h497a2fe_3    conda-forge
xorg-kbproto              1.0.7             h7f98852_1002    conda-forge
xorg-libice               1.0.10               h7f98852_0    conda-forge
xorg-libsm                1.2.3             hd9c2040_1000    conda-forge
xorg-libx11               1.7.2                h7f98852_0    conda-forge
xorg-libxau               1.0.9                h7f98852_0    conda-forge
xorg-libxdmcp             1.1.3                h7f98852_0    conda-forge
xorg-libxext              1.3.4                h7f98852_1    conda-forge
xorg-libxrender           0.9.10            h7f98852_1003    conda-forge
xorg-renderproto          0.11.1            h7f98852_1002    conda-forge
xorg-xextproto            7.3.0             h7f98852_1002    conda-forge
xorg-xproto               7.0.31            h7f98852_1007    conda-forge
xz                        5.2.5                h516909a_1    conda-forge
yaml                      0.2.5                h516909a_0    conda-forge
zeromq                    4.3.4                h9c3ff4c_1    conda-forge
zict                      2.0.0                      py_0    conda-forge
zipp                      3.5.0              pyhd8ed1ab_0    conda-forge
zlib                      1.2.11            h516909a_1010    conda-forge
zstd                      1.5.0                ha95c52a_0    conda-forge
```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info

     active environment : katana-test
    active env location : /home/amp/LocalInstalls/conda/envs/katana-test
            shell level : 2
       user config file : /home/amp/.condarc
 populated config files : /home/amp/LocalInstalls/conda/.condarc
                          /home/amp/.condarc
          conda version : 4.10.3
    conda-build version : not installed
         python version : 3.8.10.final.0
       virtual packages : __linux=5.4.0=0
                          __glibc=2.31=0
                          __unix=0=0
                          __archspec=1=x86_64
       base environment : /home/amp/LocalInstalls/conda  (writable)
      conda av data dir : /home/amp/LocalInstalls/conda/etc/conda
  conda av metadata url : None
           channel URLs : https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
          package cache : /home/amp/LocalInstalls/conda/pkgs
                          /home/amp/.conda/pkgs
       envs directories : /home/amp/LocalInstalls/conda/envs
                          /home/amp/.conda/envs
               platform : linux-64
             user-agent : conda/4.10.3 requests/2.26.0 CPython/3.8.10 Linux/5.4.0-81-generic ubuntu/20.04.3 glibc/2.31
                UID:GID : 1000:1000
             netrc file : None
           offline mode : False

```
</details>

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -26,7 +26,7 @@ requirements:
 test:
   commands:
     - test -f ${PREFIX}/include/backward.hpp  # [not win]
-    - test -f ${PREFIX}/lib/backward/BackwardConfig.cmake  # [not win]
+    - test -f ${PREFIX}/lib64/backward/BackwardConfig.cmake  # [not win]
     - if not exist %PREFIX%\\Library\\include\\backward.hpp exit 1  # [win]
     - if not exist %PREFIX%\\Library\\lib\\backward\\BackwardConfig.cmake exit 1  # [win]
 

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -12,7 +12,7 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
+  number: 2
 
 requirements:
   build:
@@ -24,6 +24,12 @@ requirements:
     - elfutils  # [linux]
 
 test:
+  commands:
+    - test -f ${PREFIX}/include/backward.hpp  # [not win]
+    - test -f ${PREFIX}/lib/backward/BackwardConfig.cmake  # [not win]
+    - if not exist %PREFIX%\\Library\\include\\backward.hpp exit 1  # [win]
+    - if not exist %PREFIX%\\Library\\lib\\backward\\BackwardConfig.cmake exit 1  # [win]
+
   requires:
     - cmake
     - {{ compiler('c') }}

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -24,12 +24,6 @@ requirements:
     - elfutils  # [linux]
 
 test:
-  commands:
-    - test -f ${PREFIX}/include/backward.hpp  # [not win]
-    - test -f ${PREFIX}/lib64/backward/BackwardConfig.cmake  # [not win]
-    - if not exist %PREFIX%\\Library\\include\\backward.hpp exit 1  # [win]
-    - if not exist %PREFIX%\\Library\\lib\\backward\\BackwardConfig.cmake exit 1  # [win]
-
   requires:
     - cmake
     - {{ compiler('c') }}

--- a/recipe/run_test.sh
+++ b/recipe/run_test.sh
@@ -2,7 +2,7 @@
 
 # Check files
 test -f ${PREFIX}/include/backward.hpp
-test -f ${PREFIX}/lib64/backward/BackwardConfig.cmake
+test -f ${PREFIX}/lib/backward/BackwardConfig.cmake
 
 # Test example build
 mkdir build

--- a/recipe/run_test.sh
+++ b/recipe/run_test.sh
@@ -1,4 +1,10 @@
 #!/bin/bash
+
+# Check files
+test -f ${PREFIX}/include/backward.hpp
+test -f ${PREFIX}/lib64/backward/BackwardConfig.cmake
+
+# Test example build
 mkdir build
 cd build
 cmake $RECIPE_DIR -DCMAKE_BUILD_TYPE=Debug

